### DISCUSSION TITLE
Android agents

### DISCUSSION THREAD

**Discussion by sharkpd** [amidaware/tacticalrmm#1280] [Ideas]
Has an Android agent been considered / planned?

**Comment by wh1te909** [comment]
not really, extremely low priority and definitely won't make it into the 1.0 official release but if someone were to start working on one I wouldn't have an issue with it

**Comment by dinger1986** [comment]
That's really moving into mdm territory.

What do you want to achieve on Android?

**Reply by sharkpd** [reply]
The helpdesk of a company may need to connect to the employee's tablet or mobile phone to verify the steps that the user takes in the face of a request for help as he does with users of windows and / or Linux PCs.

**Reply by silversword411** [reply]
Meshcentral can be used for most of that.

**Reply by sharkpd** [reply]
Of course, but the integration with the tacticrmm console makes the management of all mobile devices convenient for any operator, such as for Windows and / or Linux PCs.
